### DISCUSSION TITLE
opensubsonic IRC channel?

### DISCUSSION THREAD

**Discussion by sentriz** [opensubsonic/open-subsonic-api#59] [General]
I think for when were iterating on an extension or new feature, some instant messaging could really be beneficial

given the name opensubsonic, I propose `#opensubsonic` on liberal.chat, the home of free and open software projects

it could be open to client and server devs, and users to discuss the future of subsonic in general

**Comment by dweymouth** [comment]
sounds fine to me

**Comment by Tolriq** [comment]
The problem about IRC is that there's no history and no search, and we end up with duplicates and repeating things a lot like the v2.

So yes it's nice to start discussion, but we need to have traces here too for efficiency for the others.

**Reply by epoupon** [reply]
I do agree (and personally, I won't be able to spend time on IRC :( )

**Comment by lachlan-00** [comment]
I don't use much outside guthub or email. Won't use irc for work talk.
